sharepoint/3.1/modules/ROOT/pages/sharepoint-connector-reference.adoc

```diff
@@ -1,1320 +1,1365 @@
 
-[[fileApprove]]
-== File Approve
 
+[[fileApprove]]
+=== File Approve
 `<sharepoint:file-approve>`
 
-=== Parameters
 
+==== Parameters
 [%header,cols="20s,20a,35a,20a,5a"]
 |===
 | Name | Type | Description | Default Value | Required
-| Configuration | String | The name of the configuration to use. | |x
-| File Server Relative Url a| String | | `#[payload]` |
-| Comment a| String | | |
-| Reconnection Strategy a| * reconnect
-* reconnect-forever | A retry strategy in case of connectivity errors. | |
+| Configuration | String | The name of the configuration to use. | | x
+| File Server Relative Url a| String |  |  | x
+| Comment a| String |  |  |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors |  |
 |===
 
-=== For Configurations
 
-* sharepoint-online
-* sharepoint-onpremise
+=== Associated Configurations
+* <<sharepoint-online>>
+* <<sharepoint-onpremise>>
 
-=== Throws
-
-* SHAREPOINT:CONNECTIVITY
-* SHAREPOINT:CONNECTIVITY
+==== Throws
 * SHAREPOINT:RETRY_EXHAUSTED
 * SHAREPOINT:UNKNOWN
+* SHAREPOINT:CONNECTIVITY
 
-[[fileCheckIn]]
-== File Check In
 
+[[fileCheckIn]]
+=== File Check In
 `<sharepoint:file-check-in>`
 
-=== Parameters
 
+==== Parameters
 [%header,cols="20s,20a,35a,20a,5a"]
 |===
 | Name | Type | Description | Default Value | Required
-| Configuration | String | The name of the configuration to use. | |x
-| File Server Relative Url a| String | | `#[payload]` |
+| Configuration | String | The name of the configuration to use. | | x
+| File Server Relative Url a| String |  |  | x
 | Check In Type a| Enumeration, one of:
 
 ** MINOR_CHECK_IN
 ** MAJOR_CHECK_IN
-** OVERWRITE_CHECK_IN | | |x
-| Comment a| String | | |
-| Reconnection Strategy a| * reconnect
-* reconnect-forever | A retry strategy in case of connectivity errors. | |
+** OVERWRITE_CHECK_IN |  |  | x
+| Comment a| String |  |  | x
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors |  |
 |===
 
-=== For Configurations
-
-* sharepoint-online
-* sharepoint-onpremise
 
-=== Throws
+=== Associated Configurations
+* <<sharepoint-online>>
+* <<sharepoint-onpremise>>
 
-* SHAREPOINT:CONNECTIVITY
-* SHAREPOINT:CONNECTIVITY
+==== Throws
 * SHAREPOINT:RETRY_EXHAUSTED
 * SHAREPOINT:UNKNOWN
+* SHAREPOINT:CONNECTIVITY
 
-[[fileCheckOut]]
-== File Check Out
 
+[[fileCheckOut]]
+=== File Check Out
 `<sharepoint:file-check-out>`
 
-=== Parameters
 
+==== Parameters
 [%header,cols="20s,20a,35a,20a,5a"]
 |===
 | Name | Type | Description | Default Value | Required
-| Configuration | String | The name of the configuration to use. | |x
-| File Server Relative Url a| String | | `#[payload]` |
-| Reconnection Strategy a| * reconnect
-* reconnect-forever | A retry strategy in case of connectivity errors. | |
+| Configuration | String | The name of the configuration to use. | | x
+| File Server Relative Url a| String |  |  | x
```

Comment:
I'll just assume `payload` was intentionally removed because it is no longer the default?